CHANGELOG.md

```diff
@@ -23,10 +23,11 @@
 ## Improvements
 
 - [Generator] Generator will fail, when it generates `.ts` files that have compilation errors.
+- [OData] Add warning for deep update attempts in OData v2. 
 
 ## Fixed Issues
 
--
+- [OData] Fix handling of Etag for update requests.
```

Comment:
Maybe this is a bit too generic. Also sorry for missing this PR, but this probably belongs one section up.

Reply:
made it more specific